.github/workflows/docs-update.yml

```diff
@@ -51,15 +51,20 @@ jobs:
           git config --global user.name "${{ github.actor }}"
           git config --global user.email "${{ github.actor }}@users.noreply.github.com"
 
-      - run: poetry run mike deploy -b gh-pages dev --push
-        if: "github.ref == 'refs/heads/main'"
+      - name: Deploy on push to main
+        if: github.ref == 'refs/heads/main'
```

Comment:
The `""` was dropped here, is there a problem with that?

Reply:
I got a warning about it:

> Unexpected symbol: '"github'. Located at position 1 within expression: "github.ref == 'refs/heads/main'"
> Use the if conditional to prevent a step from running unless a condition is met. Any supported context and expression can be used to create a conditional. Expressions in an if conditional do not require the bracketed expression syntax. When you use expressions in an if conditional, you may omit the expression syntax because GitHub automatically evaluates the if conditional as an expression.
> 
> Available expression contexts: github, inputs, vars, needs, strategy, matrix, steps, job, runner, env
> 
> Available expression functions: always, failure, cancelled, success, hashFiles

Reply:
The docs say to use it under `{`, I can change that
https://docs.github.com/en/actions/learn-github-actions/contexts

![image](https://github.com/vintasoftware/django-ai-assistant/assets/22326737/2045c4d6-0d2d-43fb-aeb0-df0a96ded0a0)

Reply:
OK, please do